src/model/trainer.ts

```diff
@@ -0,0 +1 @@
+export class Trainer{}
```

Comment:
Nope, we have a main Training entity, then a TrainingSummary (just a map including a summary of the fields).  Training entity will be the one use when we want to edit a given training.